src/Config/Auth.php

```diff
@@ -184,6 +184,9 @@ class Auth extends BaseConfig
      * --------------------------------------------------------------------
      * If true, will always update the `last_active` datetime for the
      * logged in user on every page request.
+     * This feature only works when session/tokens/chain filter is set correctly.
```

Comment:
The chain filter does nothing actually, the session/tokens filter does call `recordActiveDate()`.
How about this?
```suggestion
     * This feature only works when session/tokens filter is active.
```

Reply:
Thanks. Done.